containers/freesurfer.context/freesurfer.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# docker build for distributing a base fs 7.2.0 container
+
+FROM freesurfer-builder as freesurfer
+
+ARG FREESURFER_VERSION
```

Comment:
Add an ENV definition as well here, that sets the default freesurfer version if ARG is not supplied. Refer to the other dockerfiles, they all do it.